<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [Default suite]">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="com.revature.controllertests.ContentControllerTest">
        <methods>
          <include name="springTestContextBeforeTestClass"/>
          <include name="deleteContent"/>
          <include name="setup"/>
<<<<<<< HEAD
          <include name="testSetup"/>
          <include name="tearDown"/>
          <include name="filterTest"/>
        </methods>
      </class> <!-- com.revature.servicestests.SerachServiceTest2 -->
=======
          <include name="springTestContextPrepareTestInstance"/>
          <include name="springTestContextAfterTestClass"/>
          <include name="springTestContextBeforeTestMethod"/>
          <include name="springTestContextAfterTestMethod"/>
          <include name="preTestSetup"/>
        </methods>
      </class> <!-- com.revature.controllertests.ContentControllerTest -->
>>>>>>> d8018e0519aaf766eadcfeceb64692a952607b6e
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
